src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +133,22 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
```

Comment:
Bleh, I don't like these if statements... I wonder if we could do this differently.  Ideas?